### DISCUSSION TITLE
Wrong Time in Attacker Map

### DISCUSSION THREAD

**Discussion by FoxXxHater** [telekom-security/tpotce#1363] [Q&A]
Hello Guys, 

i have in my attaker map a worng time and wrong format... how can i change that?
In the Cockpit was the right time setet.

**Comment by t3chn0m4g3** [comment]
The T-Pot Attack Map does show the time in UTC which is the timezone of T-Pot.
However this seems to confuse to some degree so I just pushed a new docker image for [AttackMap 2.1.0](https://github.com/t3chn0m4g3/t-pot-attack-map/releases/tag/2.1.0) which will recalculate the timestamp for the timezone of the browser and display the events accordingly.
